numpy/core/memmap.py

```diff
@@ -75,12 +75,17 @@ class memmap(ndarray):
         additional data. By default, ``memmap`` will start at the beginning of
         the file, even if ``filename`` is a file pointer ``fp`` and
         ``fp.tell() != 0``.
-    shape : tuple, optional
+    shape : int or sequence of ints, optional
         The desired shape of the array. If ``mode == 'r'`` and the number
         of remaining bytes after `offset` is not a multiple of the byte-size
         of `dtype`, you must specify `shape`. By default, the returned array
         will be 1-D with the number of elements determined by file size
         and data-type.
+
+        .. versionchanged:: 1.25
```

Comment:
```suggestion
        .. versionchanged:: 2.0
```
Sorry, was ready early enough really, but it slipped now.